com.github.btpf.alexandria.appdata.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.github.btpf.alexandria</id>
+
+  <name>Alexandria</name>
+  <summary>A minimalistic eBook reader</summary>
+  <developer_name>Bret Papkoff</developer_name>
+
+  <metadata_license>MIT</metadata_license>
+  <project_license>GPL-3.0+</project_license>
+
+  <url type="homepage">https://github.com/btpf/Alexandria</url>
+  <url type="bugtracker">https://github.com/btpf/Alexandria/issues</url>
+
+  <supports>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <control>touch</control>
+  </supports>
+
+  <description>
+    <p>
+      Alexandria is a minimalistic Ebook Reader supporting Epub, Kindle Format, ComicBook Format, and more
```

Comment:
Try following the quality guidelines if you can. It'll improve presentation https://docs.flathub.org/docs/for-app-authors/appdata-guidelines/quality-guidelines#not-too-short